os/fs/smartfs/smartfs_utils.c

```diff
@@ -1303,9 +1271,11 @@ int smartfs_writeentry(struct smartfs_mountpt_s *fs, struct smartfs_entry_s new_
 	struct smartfs_entry_header_s *entry;
 	struct smartfs_chain_header_s *chainheader;
 	char *tmp_buf = NULL;
+	uint16_t nextsector;
 
 	entrysize = sizeof(struct smartfs_entry_header_s) + fs->fs_llformat.namesize;
 	psector = new_entry.dsector;
+	/* In case newchain == TRUE, offset will hold the logical sector numer to which the new sector is to be chained */
```

Comment:
Old description, please remove this line or change

Reply:
Done